app/facades/quote_facade.rb

```diff
@@ -0,0 +1,10 @@
+class QuoteFacade
+
+  def self.search_term(input)
+    json = QuoteService.search_by_input(input)
+
+    @quotes = json[:results].map do |quote_data|
```

Comment:
```suggestion
    quotes = json[:results].map do |quote_data|
```
I don't think this needs to be an instance variable. You can just use a local variable `quotes` here and it'll work just as well.